### PR TITLE
Transitions

### PR DESCRIPTION
**Pull request by lucasnestrojil**
## Summary
- Added the transition effect

## Screenshots
transitions

## Trello Card
https://trello.com/c/jYYD1bYD/25-css-transitions

### CODE REVIEW

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]
Change base to previous PR

File: maquetado/setup/scss/tech.scss [hunk 1]
```diff
@@ -23,6 +23,13 @@
   }
 }
 
+.tech-container:hover {
```
Comment (mmamaniwolox) [inline maquetado/setup/scss/tech.scss]: This could be `.tech-container:hover::before`

Also, add a `transition` attribute to `tech-container` for transition to work as expected

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: maquetado/setup/scss/tech.scss [hunk 2]
```diff
@@ -8,21 +8,27 @@
...
+    top: -7px;
     z-index: -1;
     border-radius: 10px;
     width: 210px;  
     height: 300px;
+    transition: all 1s;  
   }
 }
 
+.tech-container:hover::before {
```
Comment (mmamaniwolox) [inline maquetado/setup/scss/tech.scss]: I would move this inside the `before` element as `&:hover`

File: maquetado/setup/scss/tech.scss [hunk 3]
```diff
@@ -8,21 +8,27 @@
...
-
+  align-items: center;  
+  
   &::before {
     content: '';
     border: 2px solid  $green;
     position: absolute;
-    right: 5px;
-    bottom: 5px;
+    left: -7px;
```
Comment (mmamaniwolox) [inline maquetado/setup/scss/tech.scss:17]: Why did you change it for negative values? Maybe you can use a positive `bottom` and `right`

File: maquetado/setup/scss/tech.scss [hunk 4]
```diff
@@ -8,21 +8,27 @@
...
+    top: -7px;
     z-index: -1;
     border-radius: 10px;
     width: 210px;  
     height: 300px;
+    transition: all 1s;
```
Comment (mmamaniwolox) [inline maquetado/setup/scss/tech.scss]: It will be better to select the attributes that are being modified for transition, in this case, left and top

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: maquetado/setup/scss/tech.scss (hunk 3 above)
Comment (lucasnestrojil) [reply maquetado/setup/scss/tech.scss:17]: Because i needed to use the same attributes into the transition.

File: maquetado/setup/scss/tech.scss [hunk 5]
```diff
@@ -8,18 +8,24 @@
...
     position: absolute;
-    right: 5px;
-    bottom: 5px;
+    left: -7px;
+    top: -7px;
     z-index: -1;
     border-radius: 10px;
     width: 210px;  
     height: 300px;
+    transition: left 1s, top 1s;
```
Comment (mmamaniwolox) [inline maquetado/setup/scss/tech.scss]: You should add a transition effect in here, `ease` for example

`transition: top 1s ease, left 1s ease;`

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by mmamaniwolox — APPROVED** [review]
